### DISCUSSION TITLE
Dicom2Nifti options with linux command-line

### DISCUSSION THREAD

**Discussion by nomook** [CBICA/CaPTk#1421] [Q&A] [answered]
Hello,

I'm using CaPTk in a Pipeline and for that I need to add options to dcm2niix conversion. Is there any way of adding options at that command-line ?
Such as for example :
${CaPTk_InstallDir}/captk Utilities -i /home/DicomFolder -o /home/result.nii.gz -d2n {-z Y -f %p}

Many thanks for your feedback
Noé

**Comment by sarthakpati** [comment, marked as answer]
Hi,

Thank you for your interest in CaPTk!

If you want to pass additional parameters to `dcm2niix`, you can do so directly in this manner:

```powershell
cd ${CaPTk_InstallDir}/bin # this is where all executables are present
./dcm2niix -i /home/DicomFolder -o /home/result.nii.gz -d2n {-z Y -f %p}
```

Cheers,
Sarthak

**Reply by nomook** [reply]
Well, it seems to not be the case on Linux (Ubuntu 20.04).

There is not any 'bin' folders in my CaPTk install dir.

_May help :_
To access the utilities on Linux, the command-line is the following :
`${CaPTk_InstallDir}/captk Utilities -i /home/DicomFolder -o /home/result.nii.gz -d2n`

Noé

**Reply by sarthakpati** [reply]
Ah okay. You can try the following:

```bash
${CaPTk_InstallDir}/captk --appimage-extract 
```
This should expand the installer under a new folder `squash-fs`, and traversing this directory tree should provide the `dcm2niix` executable.

**Reply by nomook** [reply]
Thank you for your speed and reliability.

Noé
